packages/metrics/src/scripts/VSExportGenerator.ts

```diff
@@ -335,12 +335,10 @@ async function setLocationInComposition(
     'Location/',
     ''
   )
-  const locationDoc = locations.find(
-    ({ id }) => id === locationId
-  ) as fhir.Location
+  const locationDoc = locations.find(({ id }) => id === locationId)
 
   const isLocationHealthFacility =
```

Comment:
fyi, I have added a "CRVS_OFFICE" type here: https://github.com/opencrvs/opencrvs-core/blob/develop/packages/commons/src/fhir/location.ts#L46

these are not your changes but maybe a meaningful refactor would create a "HealthFacility" type and `const isHealthFacility(location: Location): location is HealthFacility` helpers to commons ?

Reply:
makes perfect sense. I can amend it. Thanks, @naftis !